### PR TITLE
Adds support page

### PR DESCRIPTION
**Pull request by pedrocamargo**
@Art-Ev , what do you think?  Does this cut it?  Should we add logos and links to specific pages/blog posts/etc?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,84 +15,20 @@
 [![QAequilibraE artifacts](https://github.com/AequilibraE/aequilibrae/actions/workflows/build_artifacts_qgis.yml/badge.svg)](https://github.com/AequilibraE/aequilibrae/actions/workflows/build_artifacts_qgis.yml)
 
 
-AequilibraE is the first comprehensive Python package for transportation modeling. It aims to provide all the
+AequilibraE is a fully-featured Open-Source transportation modeling package and the first comprehensive package
+of its king for the Python ecosystem. It provides all the most fundamental
```
Comment (jamiecook) [inline README.md]:
```suggestion
of its kind for the Python ecosystem. It aims to provide all the fundamental transport modelling
```

File: README.md [hunk 2]
```diff
@@ -15,84 +15,20 @@
...
-* GTFS Import
 
 ### What is available only in QGIS
 
 Some common resources for transportation modeling are inherently visual, and therefore they make more sense if
 available within a GIS platform. For that reason, many resources are available only from AequilibraE's 
 [QGIS plugin](http://plugins.qgis.org/plugins/qaequilibrae/),
 which uses AequilibraE as its computational workhorse and also provides GUIs for most of AequilibraE's tools. Said tool
-is developed independently, although in parallel, and more details can be found in its 
+is developed independently and a little delayed with relationship to the Python package, and more details can be found in its
```
Comment (jamiecook) [inline README.md:33]:
```suggestion
is developed independently and new features from Aequilibrae may not be immediately available through the QGIS interface. More details can be found in its 
```

File: docs/source/support.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+1. Joining the `AequilibraE Google Group <https://groups.google.com/forum/#!forum/aequilibrae>`_
+   and sending your question there.
+2. Posting your question to `GIS StackOverflow <https://gis.stackexchange.com/>`_ using the #aequilibrae tag;
+
+Please note that all questions and answers in both forums are public.
+
+Paid support
+------------
+
+Paid support for AequilibraE is offered by Outer Loop Consulting, and Australia-based consulting company,
```
Comment (jamiecook) [inline docs/source/support.rst]:
```suggestion
Paid support for AequilibraE is offered by Outer Loop Consulting, an Australia-based consulting company,
```

File: README.md [hunk 4]
```diff
@@ -15,84 +15,20 @@
...
 Some common resources for transportation modeling are inherently visual, and therefore they make more sense if
 available within a GIS platform. For that reason, many resources are available only from AequilibraE's 
 [QGIS plugin](http://plugins.qgis.org/plugins/qaequilibrae/),
 which uses AequilibraE as its computational workhorse and also provides GUIs for most of AequilibraE's tools. Said tool
-is developed independently, although in parallel, and more details can be found in its 
+is developed independently and a little delayed with relationship to the Python package, and more details can be found in its 
 [GitHub repository](https://github.com/AequilibraE/qaequilibrae).
-
-
-### What is not planned to be available any time soon
```
Comment (jamiecook) [inline README.md:44]: Is this not worth keeping?

**Review by jamiecook — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (pedrocamargo) [reply README.md:44]: These things fall through the cracks... Until the other day, PT assignment was in the list...    I think that a proper roadmap will solve this issue.

**Pull request merged**